etc/kayobe/ansible/redfish-exporter.yml

```diff
@@ -0,0 +1,35 @@
+---
+- hosts: monitoring
+  gather_facts: false
+  tasks:
+   - copy:
+       src: "{{playbook_dir}}/dockerfiles/redfish-exporter.Dockerfile"
+       dest: /opt/kayobe/images/redfish-exporter.Dockerfile
+
+   - copy:
```

Comment:
Is this world readable?

Reply:
It is, I saw that and forgot, nice catch. Although the directory is protected, I should add a mode on this file.

Reply:
Could pass creds in as build args?